docs/freqai-data-handling.md

```diff
@@ -0,0 +1,115 @@
+# Data handling
```

Comment:
I find this "running FreqAI" and "data handling" segments quite confusing / disorganized.

I think the best way would be to move the "filestructure" part to the developer page - and the rest to running freqAI.

That's based on the assumption that i'm new here to play around freqAI - and i'm looking for "how do i run it" - so "running freqAI" is the place to go. Unfortunately, it only includes half of the information (it's missing model purging, for example)

Reply:
Agreed. Moving 'file structure' to the Dev page, 'Returning data from training' to 'Feature engineering', and the rest to 'Running FreqAI'.